### PR TITLE
[CI] Merge Distributed and Linux GPU job

### PR DESCRIPTION
**Pull request by osalpekar**
Runs the distributed tests as part of the Linux CPU/GPU jobs. Confirmed the distributed tests are passing. Failing tests seen on this PR for the Linux jobs are also failing on master.

### CODE REVIEW

**Comment by osalpekar** [comment]
cc @vmoens

File: .circleci/unittest/linux/scripts/run_test.sh [hunk 1]
```diff
@@ -21,6 +21,6 @@ export CKPT_BACKEND=torch
 
 python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest test/smoke_test.py -v --durations 200
 python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest test/smoke_test_deps.py -v --durations 200 -k 'test_gym or test_dm_control_pixels or test_dm_control or test_tb'
-python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest --instafail -v --durations 200 --ignore test/test_distributed.py --ignore test/test_rlhf.py
+python .circleci/unittest/helpers/coverage_run_parallel.py -m pytest --instafail -v --durations 200
```
Comment (vmoens) [inline .circleci/unittest/linux/scripts/run_test.sh:24]: (this file isn't used anymore I believe)
test_rlhf should still be excluded

**Review by vmoens — APPROVED** [review]
LGTM
Don't we want to also deprecate the distributed workflows while we're at it?

File: .circleci/unittest/linux/scripts/run_test.sh (hunk 1 above)
Comment (osalpekar) [reply .circleci/unittest/linux/scripts/run_test.sh:24]: Yeah we can probably just delete this file in a follow-up since it's not used anymore.

**Pull request merged**